presidio-analyzer/presidio_analyzer/entity_recognizer.py

```diff
@@ -54,11 +54,15 @@ def __init__(
         self.version = version
         self.is_loaded = False
         self.context = context if context else []
+        self.supported_entities = supported_entities
 
         self.load()
         logger.info("Loaded recognizer: %s", self.name)
         self.is_loaded = True
 
+        if analyzer_utils is not None:
+            self.analyzer_utils = analyzer_utils
```

Comment:
class fields should not be optional. I would suggest to remove the `if analyzer_utils is not None` and put the value anyway (even if it's None). Otherwise the `analyzer_utils` field would not always be part of the class.